drivers/flash/jesd216.h

```diff
@@ -12,6 +12,11 @@
 #include <zephyr/sys/util.h>
 #include <zephyr/types.h>
 
+/* JEDEC Read identification */
+#define JESD216_CMD_READ_ID   0x9F
```

Comment:
Duplicates https://github.com/zephyrproject-rtos/zephyr/blob/0514c3b139bffd1813e862a1de42aece0f454c1e/drivers/flash/spi_nor.h#L45

Reply:
To remove duplicate, it could be 
`#define JESD216_CMD_READ_ID   SPI_NOR_CMD_RDID`